socket/index.js

```diff
@@ -85,18 +169,26 @@ io.on('connection', (socket) => {
 		if (err) console.error(err);
 	  });
 
-	  Session.findOne( { id: data.session }, (err, sessions) => {
-		if (err) console.error(err);
-		
-		sessions.users.splice(sessions.users.indexOf(data.user), 1);
-		sessions.save();
-	  })
-
-	  currSession = allUsers[data.session]
-	  delete currSession[data.user]
+	  Session.update( 
+		  { id: data.session },
+		  { $pull: { users : { _id : data.user } } },
+		  { safe: true },
+		  (err, obj)  => {});
+		  
+		// Delete user from session
+		currSession = allUsers[data.session]
```

Comment:
Need to make sure you are ending statements with a `;`. ESLint will catch these :-)